src/epydeck/__init__.py

```diff
@@ -47,7 +49,7 @@ def _parse_block(line: str, fh: TextIOBase) -> dict:
             break
 
         # Handle special keywords
-        if line.lower().startswith("include_species:"):
+        if any(line.lower().startswith(keyword + ":") for keyword in special_keywords):
```

Comment:
Better to use an f-string to build strings:

```suggestion
        if any(line.lower().startswith(f"{keyword}:") for keyword in special_keywords):
```